syz-manager/manager.go

```diff
@@ -118,13 +197,31 @@ type Manager struct {
 	benchMu   sync.Mutex
 	benchFile *os.File
 
-	assetStorage *asset.Storage
-
-	reproMgr *reproManager
-
 	Stats
 }
 
+func newManager(
```

Comment:
Fold whole function signature onto a single line, it does not look too long for splitting.